burn-tensor/src/tensor/data.rs

```diff
@@ -206,12 +252,28 @@ where
 
         Data::new(data, shape)
     }
+
+    /// Populates the data with ones.
+    ///
+    /// # Arguments
+    ///
+    /// * `shape` - The shape of the data.
+    /// * `_kind` - The kind of the data.
+    ///
+    /// # Returns
+    ///
+    /// The data with ones.
     pub fn ones_(shape: Shape<D>, _kind: E) -> Data<E, D> {
         Self::ones(shape)
     }
```

Comment:
I think we should remove those functions (`ones_`, `zeros_`) that accept an unused kind as input.